pepper-apis/src/main/java/org/broadinstitute/ddp/content/SelectedPickListOptionRenderUtil.java

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.ddp.content;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.broadinstitute.ddp.model.activity.definition.question.PicklistOptionDef;
+import org.broadinstitute.ddp.model.activity.definition.question.PicklistQuestionDef;
+import org.broadinstitute.ddp.model.activity.definition.question.QuestionDef;
+import org.broadinstitute.ddp.model.activity.instance.answer.Answer;
+import org.broadinstitute.ddp.model.activity.instance.answer.PicklistAnswer;
+import org.broadinstitute.ddp.model.activity.instance.answer.SelectedPicklistOption;
+import org.broadinstitute.ddp.model.activity.instance.question.PicklistOption;
+import org.broadinstitute.ddp.model.activity.instance.question.PicklistQuestion;
+import org.broadinstitute.ddp.model.activity.instance.question.Question;
+
+
+/**
+ * Utility methods for rendering selected picklist options and picklist option detail text.
+ */
+public class SelectedPickListOptionRenderUtil {
+
+    public static String selectedOptionLabelsRender(QuestionDef questionDef, Answer answer, String fallbackValue, String isoLangCode) {
+        Map<String, PicklistOptionDef> options = ((PicklistQuestionDef) questionDef)
+                .getAllPicklistOptions().stream()
+                .collect(Collectors.toMap(PicklistOptionDef::getStableId, Function.identity()));
+        return ((PicklistAnswer) answer).getValue().stream()
+                .map(selected -> renderOptionLabelOrFallback(options, selected, fallbackValue, isoLangCode))
```

Comment:
Rendering option label OR fallback is different from current behavior. Currently, we don't use the fallback in this way. Notice that in `RenderValueProvider`, the fallback is used if there is no answer for the question (specifically we check `answer.isEmpty()`). So by the time do get to this rendering here, we're guaranteed that there's at least one selected option. Also, the option_stable_id should match up, otherwise, it means we've selected an option that's not part of the question, and that will mean there's something else wrong here. So my point here is that, we shouldn't need to deal with the fallback here. In fact, if we used fallback here, it might be hiding a potential bug.

Reply:
Fixed.